packages/contractkit/src/utils/azure-key-vault-client.ts

```diff
@@ -192,8 +186,13 @@ export class AzureKeyVaultClient {
   private static bufferToBigNumber(input: Buffer): BigNumber {
     return new BigNumber(ensureLeading0x(input.toString('hex')))
   }
-  private static bigNumberToBuffer(input: BigNumber): Buffer {
-    return ethUtil.toBuffer(ensureLeading0x(input.toString(16))) as Buffer
+  private static bigNumberToBuffer(input: BigNumber, size: number): Buffer {
```

Comment:
@codyborn I wonder if we should move `bigNumberToBuffer` and `bufferToBigNumber` into a utils file later on

Reply:
Agreed. We can do this in a follow up PR.